manifests/init.pp

```diff
@@ -55,6 +55,9 @@
 # @param rules A hash that holds all data for ferm::rule
 #   Default value: Empty Hash
 #   Allowed value: Any Hash
+# @param chains A hash that holds all data for ferm::chain
+#   Default value: Empty Hash
```

Comment:
Default values and allowed values are produced by `puppet strings` from the puppet code.

Reply:
which rake task do I need to run in order to generate them?